cmd/cp-main.go

```diff
@@ -93,11 +93,11 @@ EXAMPLES:
    4. Copy a bucket recursively from aliased Amazon S3 cloud storage to local filesystem on Windows.
       $ {{.HelpName}} --recursive s3\documents\2014\ C:\Backups\2014
 
-   5. Copy files older than 7 days from Minio cloud storage to Amazon S3 cloud storage.
-      $ {{.HelpName}} --older-than 7 play/mybucket/burningman2011/ s3/mybucket/
+   5. Copy files older than 7 days and 10 hours from Minio cloud storage to Amazon S3 cloud storage.
+      $ {{.HelpName}} --older-than "7d10h" play/mybucket/burningman2011/ s3/mybucket/
 
-   6. Copy files newer than 7 days from Minio cloud storage to a local path.
-      $ {{.HelpName}} --newer-than 7 play/mybucket/burningman2011/ ~/latest/
+   6. Copy files newer than 7 days and 10 hours from Minio cloud storage to a local path.
+      $ {{.HelpName}} --newer-than "7d10h" play/mybucket/burningman2011/ ~/latest/
```

Comment:
Also, it looks like we do not have to have double quotes around the value - `7d10h` seems to work.